<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="NombreLabel">
        <property name="text">
         <string>Nombre:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="NombreEdit"/>
      </item>
      <item>
       <widget class="QLabel" name="CorreoLabel">
        <property name="text">
         <string>Correo:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="CorreoEdit"/>
      </item>
      <item>
       <widget class="QLabel" name="TelefonoLabel">
        <property name="text">
         <string>Telefono:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="TelefonoEdit"/>
      </item>
      <item>
       <widget class="QLabel" name="ActividadLabel">
        <property name="text">
         <string>Usted Prefiere ir a un restaurante a:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="ComidaComboBox"/>
      </item>
      <item>
       <widget class="QLabel" name="DiaLabel">
        <property name="text">
         <string>Y acostumbra a ir el dia:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="DiaComboBox"/>
      </item>
      <item>
       <widget class="QLabel" name="ComidaPreferidaLabel">
        <property name="text">
         <string>Comida Preferida:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextEdit" name="ComidaPreferidaEdit"/>
      </item>
      <item>
       <widget class="QLabel" name="verLabel">
        <property name="text">
         <string>Ver Informacion:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextEdit" name="InformacionEdit"/>
      </item>
      <item>
       <widget class="QPushButton" name="enviarButton">
        <property name="text">
         <string>Enviar</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="verButton">
        <property name="text">
         <string>Ver Informacion</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>ComidaComboBox</tabstop>
  <tabstop>CorreoEdit</tabstop>
  <tabstop>TelefonoEdit</tabstop>
  <tabstop>NombreEdit</tabstop>
  <tabstop>InformacionEdit</tabstop>
  <tabstop>ComidaPreferidaEdit</tabstop>
  <tabstop>DiaComboBox</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
